starter/README.md

```diff
@@ -0,0 +1,81 @@
+# Base Starter for Vaadin Flow and Karaf in NPM mode
+
+The simplest way to start the project is run command `mvn install` to install project artifact and
+`mvn install -Prun` to run Karaf OSGi container.
+
+Note that it needs some time to deploy and start all the features so it may take some time when the URL 
+http://localhost:8181 starts to work.
+
+The command `mvn install` may be used to only build the project without running Karaf.
+This command also builds a Karaf feature "project-base-karaf" which contains all the 
+required bundles/dependencies to be able to start the project bundle.
+
+If you have a Karaf container already installed then you may install and run the project 
+via Karaf interactive shell:
+
+```
+./bin/karaf // starts interactive Karaf shell
+karaf@root()> feature:install http
+karaf@root()> feature:install war  // this and above commands starts PAX-web Http Whiteboard implementation
+karaf@root()> feature:repo-add mvn:com.example/project-base-karaf/1.0-SNAPSHOT/xml/features //adds project-base feature repository generated by maven plugin
+karaf@root()> feature:install project-base-karaf // installs and starts project feature
+```
+
+Once the commands are completed (it may take some time) open the URL http://localhost:8181 in your browser.
+
+## Servlet configuration
+
+Vaadin Servlet initial parameters may be passed via `@Component` annotation parameter `property`, e.g.
+
+```java
+@Component(immediate = false, service = Servlet.class, property = {
+  HttpWhiteboardConstants.HTTP_WHITEBOARD_SERVLET_INIT_PARAM_PREFIX +
+  InitParameters.SERVLET_PARAMETER_PRODUCTION_MODE + "=false"})
+```
+or via some annotation like `VaadinMode` which is available in the example.
+
+Another way to set the `productionMode` value is explicitly set it via the Vaadin maven plugin configuration:
+
+```xml
+ <configuration>
+    <productionMode>false</productionMode>
+</configuration>
+
+```
```

Comment:
```suggestion
```xml
<configuration>
    <productionMode>false</productionMode>
</configuration>
```
```
Minor formatting mistake on the example